src/GraphNodeManagerViewExtension/Converters/GraphNodeManagerConverter.cs

```diff
@@ -88,7 +91,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
             var infoType = parameter as string;
             if (infoType == null) return Visibility.Collapsed;
 
-            if ((ElementState)value == ElementState.Info && parameter.Equals("Info"))
+            if ((ElementState)value == ElementState.Info && parameter.Equals("Info") || (ElementState)value == ElementState.PersistentInfo && parameter.Equals("PersistentInfo"))
```

Comment:
I know you following the example here - is it possible to use nameof or something else that is less brittle though? Using constants and nameof etc?

Reply:
changed to nameof